tests/unit/python/foglamp/common/test_jqfilter.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+""" Test common/jqfilter.py
+
+"""
+
+import pytest
+import pyjq
+from unittest.mock import patch
```

Comment:
standard import "from unittest.mock import patch" should be placed before "import pytest"